lib/tasks/migrate/20220512132847_reindex_alegre.rake

```diff
@@ -0,0 +1,79 @@
+namespace :check do
+  namespace :migrate do
+    desc "Updates ProjectMedia titles and descriptions on alegre's index"
+    task update_alegre_stored_project_media: :environment do
+      started = Time.now.to_i
+      running_bucket = []
+      log_errors = []
+      temp_ids = []
+      team_total = BotUser.alegre_user.team_bot_installations.count
+      counter = 0
+      sent_cases = []
+      received_cases = []
+      BotUser.alegre_user.team_bot_installations.find_each do |tb|
```

Comment:
We can skip teams that don't have text similarity enabled: `next if !tb.team.get_master_similarity_enabled || !tb.team.get_text_similarity_enabled`